.github/workflows/benchmarks.yml

```diff
@@ -12,8 +12,9 @@ on:
     paths:
       - 'pygmt/**/*.py'
       - '.github/workflows/benchmarks.yml'
-  # Uncomment the 'pull_request' line below to trigger the workflow in PR
-  # pull_request:
+  # Run in PRs but only if the PR has the 'run/benchmark' label
```

Comment:
Should we document in `.github/PULL_REQUEST_TEMPLATE.md` or somewhere for contributors to request that the `run/benchmark` label gets added to PRs that might affect performance? Or just rely on the maintainers to set the label for now?

Reply:
I prefer to leave the jobs to the maintainers. We probably will document it in the contributors guides when addressing https://github.com/GenericMappingTools/pygmt/issues/1113.

Reply:
Ok, or we can put it in `doc/maintenance.md` somewhere. Not urgent though, since we aren't running the benchmarks that often.